### ISSUE TITLE
ClientStatic.ts does not pass noImplicitAny checks with TSC. Simple fix.

### ISSUE THREAD

**Issue by opgbaudouin** [contensis/contensis-management-api#17]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `contensis-management-api@1.0.2-beta.2` for the project I'm working on.

when using the code in a pure typescript environment using TSC (typescript 4.2.3) using the 'strict:true' is TJSON there is 1 file that has an 'implicitAny return'. All it needed to pass is add :any . (or naturally if it does return something to add that).

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/contensis-management-api/src/models/ClientStatic.ts b/node_modules/contensis-management-api/src/models/ClientStatic.ts
index da771f3..62581f9 100644
--- a/node_modules/contensis-management-api/src/models/ClientStatic.ts
+++ b/node_modules/contensis-management-api/src/models/ClientStatic.ts
@@ -3,5 +3,5 @@ import { Config } from './Config';
 export interface ClientStatic {
 	defaultClientConfig: Config;
 	create(config?: Config): ContensisClient;
-	configure(config: Config);
+	configure(config: Config): any;
 }
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by alexpop-zengenti** [comment]
Hi Onno, 
We TypeScript 3 at the moment but we plan to upgrade to TypeScript 4 and we will address this issue together with similar problems during the upgrade.

**Comment by alexpop-zengenti** [comment]
Hi Onno, this issue should be solved in our 2.1.0 release.
Cheers,
Alex
